AsyncDisplayKit/ASTableView.mm

```diff
@@ -159,7 +159,8 @@ void ASPerformBlockWithoutAnimation(BOOL withoutAnimation, void (^block)()) {
 
 - (void)configureWithAsyncDataFetching:(BOOL)asyncDataFetchingEnabled
 {
-  _layoutController = [[ASFlowLayoutController alloc] initWithScrollOption:ASFlowLayoutDirectionVertical];
+  UICollectionViewFlowLayout *flowLayout = [[UICollectionViewFlowLayout alloc] init];
```

Comment:
Super interesting that flow layout works without a collection view! Does this flow layout need to know the sizes of the table view cells? Flow layout does have an itemSize property. I'm wondering if this will only work for table views with uniform cell size since flow layout only supports custom cell sizing via collection view delegate and we don't have a collection view to attach a delegate.

If we find that using flow layout isn't sufficient for ASTableView and we want to use ASCollectionViewLayoutController, I think we could build a custom collection view layout that uses a table view to calculate layout attributes. I imagine most of that logic already lives in ASFlowLayoutController. The trick would be coordinating table view data source changes with the layout.